src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

```diff
@@ -466,26 +474,29 @@ public sealed partial class SqlConnection : System.Data.Common.DbConnection, Sys
         public SqlConnection() { }
         public SqlConnection(string connectionString) { }
         public SqlConnection(string connectionString, System.Data.SqlClient.SqlCredential credential) { }
+        public string AccessToken { get { throw null; } set { } }
         public System.Guid ClientConnectionId { get { throw null; } }
-        object ICloneable.Clone() { throw null; }
         public override string ConnectionString { get { throw null; } set { } }
         public override int ConnectionTimeout { get { throw null; } }
+        public System.Data.SqlClient.SqlCredential Credential { get { throw null; } set { } }
         public override string Database { get { throw null; } }
         public override string DataSource { get { throw null; } }
+        protected override System.Data.Common.DbProviderFactory DbProviderFactory { get { throw null; } }
         public bool FireInfoMessageEventOnUserErrors { get { throw null; } set { } }
         public int PacketSize { get { throw null; } }
         public override string ServerVersion { get { throw null; } }
         public override System.Data.ConnectionState State { get { throw null; } }
         public bool StatisticsEnabled { get { throw null; } set { } }
         public string WorkstationId { get { throw null; } }
-        public System.Data.SqlClient.SqlCredential Credential { get { throw null; } set { } }
         public event System.Data.SqlClient.SqlInfoMessageEventHandler InfoMessage { add { } remove { } }
         protected override System.Data.Common.DbTransaction BeginDbTransaction(System.Data.IsolationLevel isolationLevel) { throw null; }
         public new System.Data.SqlClient.SqlTransaction BeginTransaction() { throw null; }
         public new System.Data.SqlClient.SqlTransaction BeginTransaction(System.Data.IsolationLevel iso) { throw null; }
         public System.Data.SqlClient.SqlTransaction BeginTransaction(System.Data.IsolationLevel iso, string transactionName) { throw null; }
         public System.Data.SqlClient.SqlTransaction BeginTransaction(string transactionName) { throw null; }
         public override void ChangeDatabase(string database) { }
+        public static void ChangePassword(string connectionString, System.Data.SqlClient.SqlCredential credential, System.Security.SecureString newSecurePassword) { }
```

Comment:
This rename is a breaking change. Please let me know whether the source needs to change or not.
@afsanehr, @tarikulsabbir, @david-engel

Reply:
https://referencesource.microsoft.com/#System.Data/System/Data/SqlClient/SqlConnection.cs,16dcc922d7ff5522 suggests `newSecurePassword` is the right answer (so the ref is now right, and the source already was)